src/anonymous-credentials/schema.ts

```diff
@@ -972,6 +982,10 @@ export class CredentialSchema extends Versioned {
     return min >= 0 ? { type: this.POSITIVE_INT_TYPE } : { type: this.INT_TYPE, minimum: min };
   }
 
+  static parseBooleanType(node: { minimum?: number }, parsingOpts: ISchemaParsingOpts, nodeName: string): object {
```

Comment:
Remove `{ minimum?: number }`

Reply:
might be worth looking at integrating a linter into CI for easier catching of these, i know there is prettier but doesnt always give errors on unused fields etc. this was copy/pasted from other method to begin with.

i did want to ask though, do we need to supply a minimum at all? assuming not as it seems to work fine without it, maybe minimum of 0 makes sense if it actually changes anything?